sacrebleu.py

```diff
@@ -1549,9 +1549,11 @@ def main():
         logging.error('and you cannot then provide multiple reference files.')
         sys.exit(1)
 
-    if args.test_set is not None and args.test_set not in DATASETS:
-        logging.error('Unknown test set "%s"\n%s', args.test_set, get_a_list_of_testset_names())
-        sys.exit(1)
+    if args.test_set is not None:
+        for test_set in args.test_set.split(','):
+            if test_set not in DATASETS:
+                logging.error('Unknown test set "%s"\n%s', test_set, get_a_list_of_testset_names())
+                sys.exit(1)
 
```

Comment:
How about adding `nargs='+'` to `args.test_set`, in which case it would already be a list? I think I prefer this, too, since then the test sets can follow the param as a space-delimited list, instead of comma-delimited.

Reply:
Personally, I prefer comma-delimited over space-delimited for this purpose, but I will change this if you really think it is better. Note that:
* The semantics of `-m bleu chrf` is that we want to do two separate evaluations (and we are lazy to write a bash for cycle). However, the semantics of `-t wmt18,wmt19` is not to report two BLEU scores, but to report a single BLEU on the concatenation. Originally, I chose the plus symbol (`-t wmt18+wmt18`) instead of comma to make this explicit, but it clashes with the plus symbols in signature. Ampersand symbol is not bash friendly, so in the end I chose the comma.
* Some users prefer the style with long option names and the equal symbol (`--option=value`), which is not compatible with `nargs='+'`. Currently, you can write `--test-set=wmt18,wmt19`, but `--test-set=wmt18 wmt19` would not work (similarly as `--metrics=bleu chrf` does not work, which annoys me a bit).
* We probably don't want to use spaces in the signature, so we would have to come up with some symbol anyway. It seems a bit confusing to use comma (or whatever) in the signature, but force users to use space on the command line.

Reply:
Okay, you convinced me on this :)